specification.swagger.yaml

```diff
@@ -597,133 +48,229 @@ paths:
+            Prefer the *services* parameter unless service nodes are
+            specifically needed.
+          schema:
+            type: array
+            items:
+              type: integer
+          example: 2125,1090
+        - name: exclude_service_nodes
           in: query
-          type: string
-          description: A comma-separated list of service node ids to be used as an exclusion filter
+          description: A comma-separated list of service node ids to be used as an
+            exclusion filter
+          schema:
+            type: array
+            items:
+              type: integer
+          example: 2125,1090
         - name: division
           in: query
-          type: string
-          description: A comma-separated list of administrative divisions to be used as a filter. Use either full division ids or shorthands of the form muni/type\:id
+          description: A comma-separated list of administrative divisions to be used as a
+            filter. Use either full division ids or shorthands of the form
+            muni/type\:id
           required: false
-        - $ref: "#/parameters/unit_bbox_param"
+          schema:
+            type: array
+            items:
+              type: string
+          example: helsinki/kaupunginosa:kallio
+        - $ref: "#/components/parameters/unit_bbox_param"
         - name: bbox_srid
           in: query
-          type: string
-          description: An SRID coordinate reference system identifier which specifies the coordinate system used in the bbox parameter
+          description: An SRID coordinate reference system identifier which specifies the
+            coordinate system used in the bbox parameter
+          schema:
+            type: integer
+          example: 3046
         - name: geometry
           in: query
-          type: string
-          description: If parameter value is either true or 1, return the complex geometry of the unit in the field "geometry". Most units only have point type geometries, which are returned in the location field. Some however have polygon type geometries, eg. skiing tracks or outdoor areas. These complex geometries must be explicitly requested with this parameter.
-        # - name: maintenance_organization FIXME: no in use yet -- awaiting observations
-        #   in: query
-        #   type: string
-        #   description: A highly specialized query parameter used for filtering based on the
-
+          description: If parameter value is either true or 1, return the complex geometry
+            of the unit in the field "geometry". Most units only have point type
+            geometries, which are returned in the location field. Some however
+            have polygon type geometries, eg. skiing tracks or outdoor areas.
+            These complex geometries must be explicitly requested with this
+            parameter.
+          schema:
+            # FIXME only this values are that matters?
+            oneOf:
```

Comment:
Perhaps it would be better to just document this as boolean ("true"/"false"). I think one/zero is more of an implementation detail?

Also OpenAPI supports default values, defined as "the value server uses when one is not specified". This parameter would be an excellent candidate for that.
ie.
`- type: boolean
  example: true
  default: false`

Reply:
There will be still filter in endpoint which will be accepting values true and 1 and validation are based on those. Should this update be included in smbackend version 3 or is it safe to change validation already for this version?
Or should implementation stay as it is?

Reply:
Implementation should stay as it is. I think docs should offer the preferred value.

Reply:
check